consensus/hotstuff/validator/validator_test.go

```diff
@@ -316,7 +316,7 @@ func (vs *VoteSuite) TestVoteSignatureInvalid() {
 
 	// make sure the signature is treated as invalid
 	*vs.verifier = mocks.Verifier{}
-	vs.verifier.On("VerifyVote", vs.signer, vs.vote.SigData, vs.block).Return(false, nil)
+	vs.verifier.On("VerifyVote", vs.signer, vs.vote.SigData, vs.block).Return(fmt.Errorf("staking sig is invalid: %w", model.ErrInvalidSignature))
```

Comment:
following up on [Yurii's comment](https://github.com/onflow/flow-go/pull/1741/files#r766681102), I think we should _replace_  the assertion at the end of the test https://github.com/onflow/flow-go/blob/344e69f50ea374cabac8abd5ab84a179f51baa69/consensus/hotstuff/validator/validator_test.go#L323 with
```golang
assert.True(vs.T(), model.IsInvalidVoteError(err), "a vote with an invalid signature should be rejected")
```